pkg/digger/digger.go

```diff
@@ -247,11 +248,16 @@ func ConvertGithubEventToCommands(event models.Event, impactedProjects []configu
 				})
 			}
 		}
-		return commandsPerProject, nil
+		return commandsPerProject, true, nil
 	case models.IssueCommentEvent:
 		event := event.(models.IssueCommentEvent)
 		supportedCommands := []string{"digger plan", "digger apply", "digger unlock", "digger lock"}
 
+		if requestedProject != nil {
```

Comment:
If think `&& requestedProject.Name == impactedProjects[0].Name` is automatically implied if `len(impactedProjects) == 1`

Instead maybe log an error if ` requestedProject.Name != impactedProjects[0].Name` which would mean there is a bug..

Reply:
I wonder if failing fast is still better

Reply:
defensive programming :D

Reply:
I understand. There probability of an issue related to this is low.  Let's keep it as is.